brainstormforce/astra
# AST-2760 - Few cases for editor page title font weight mismatch with frontend.

### Description

- When font family is changed for dynamic page title, then some font weight may default to inherit, handling those cases.
- JIRA - https://brainstormforce.atlassian.net/browse/AST-2760
### Screenshots

https://d.pr/i/phe81Y

### Types of changes


Fix: When font family is changed for dynamic page title, then some font weight may default to inherit, causing editor page title to mismatch.



### How has this been tested?

- Choose any font family which defaults to 'inherit' in dynamic page title customizer.
- Reload editor, reload frontend.
- Both places font weight should be same.
### Checklist:
